### PR TITLE
Calculation logic have been added

### PR DESCRIPTION
**Pull request by misterpaul4**
operate and calculate methods was added to handle the main calculation logic.

### CODE REVIEW

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  return modObj;
+};
+
+// const testObj = {
+//   total: '5',
+//   next: '',
+//   operation: '+',
+// };
+
+// console.log(calculate(testObj, '.'));
```
Comment (bellom) [inline src/logic/calculate.js]: It’s better to delete commented code. If you need to review a previous version of your code, use git history. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    case '-':
+      return x.minus(y).toNumber();
+    case '+':
+      return x.plus(y).toNumber();
+    case 'x':
+      return x.times(y).toNumber();
+    case '/':
+      return x.div(y).toNumber();
+    case '%':
+      return x.div(100).toNumber();
```
Comment (bellom) [inline src/logic/operate.js:18]: According to the project requirement, I think you need to use `modifier` here.

image
https://microverse.pathwright.com/library/react-redux-v11/184438/path/step/106376018/

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @misterpaul4 :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: src/logic/operate.js (hunk 2 above)
Comment (misterpaul4) [reply src/logic/operate.js:18]: i did not understand what it meant

**Review by wathigo — APPROVED** [review]
## Approved :heavy_check_mark:  
Great job implementing the suggested changes.

**You may move on.**

**Happy Coding :computer: 
Simon Wathigo**

File: src/logic/operate.js (hunk 2 above)
Comment (bellom) [reply src/logic/operate.js:18]: @misterpaul4,

It is one of the `Big.js` features.
kindly read more 👉 https://mikemcl.github.io/big.js/

Example: `x.mod(y).toString()`

**Pull request merged**